src/pages/legal/privacy.jsx

```diff
@@ -0,0 +1,151 @@
+var React = require('react/addons');
+var Footer = require('../../components/footer/footer.jsx');
+
+module.exports = React.createClass({
+  mixins: [
+    React.addons.LinkedStateMixin
+  ],
+  render: function () {
+    return (
+      <div id="splash">
+        <div id="masthead">
```

Comment:
same bit as error.jsx, let's make the masthead its own component and keep things DRY